corehq/apps/cloudcare/static/cloudcare/js/formplayer/spec/fixtures/split_screen_case_list.js

```diff
@@ -34,6 +34,9 @@ hqDefine("cloudcare/js/formplayer/spec/fixtures/split_screen_case_list", functio
                 "details": null,
             },
         ],
+        "groupHeaders": {
```

Comment:
this should be inside `queryResponse`

Reply:
I just put it in both because it can be in either

Reply:
What do you mean by that? It isn't directly in the entity response. It would only be nested the `queryResponse` and the `queryResponse` would be added to the entity response.